src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1037,7 +1037,7 @@ public function getPort()
             $pos = strrpos($host, ':');
         }
 
-        if (false !== $pos) {
+        if (false !== $pos && !empty(substr($host, $pos + 1))) {
```

Comment:
I would write something like this:

```php
if (false !== $pos && '' !== $port = substr($host, $pos + 1)) {
    return (int) $port;
}
```

Reply:
Does this follow the normal style of Symfony? I haven't contributed much to Symfony before, and not sure what they prefer in terms of readability.

Can I get a second opinion on this before I make the change?

Reply:
I confirm, we avoid using empty when possible.

Reply:
Ok great, I'll make that change now.

Reply:
Change made.